src/components/EthPriceSimple.tsx

```diff
@@ -0,0 +1,106 @@
+"use client"
+
+import { useEffect, useState } from "react"
+import { Info } from "lucide-react"
+import { useLocale } from "next-intl"
+
+import type { LoadingState } from "@/lib/types"
+
+import Tooltip from "@/components/Tooltip"
+import InlineLink from "@/components/ui/Link"
+
+import { cn } from "@/lib/utils/cn"
+
+import { useTranslation } from "@/hooks/useTranslation"
+
+type EthPriceResponse = {
+  ethereum: {
+    usd: string
+  }
+}
+
+type EthPriceState = {
+  currentPriceUSD: string
+}
+
+const EthPriceSimple = ({
+  className,
+  ...props
+}: React.HTMLAttributes<HTMLDivElement>) => {
+  const locale = useLocale()
+  const { t } = useTranslation()
+  const [state, setState] = useState<LoadingState<EthPriceState>>({
+    loading: true,
+  })
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch(
+          "https://api.coingecko.com/api/v3/simple/price?ids=ethereum&vs_currencies=usd"
```

Comment:
This is because we need real-time price? aren't we going to hit rate limits pretty fast?

would recommend passing the value as a prop, fetch this at the page level (as the other data fetching calls) and maybe revalidate the page hourly.